<launch>
    <arg name="model_name" default="kingfisher"/>
    <arg name="config_pkg" default="$(eval find(arg('model_name') + '_description'))"/>

    <node pkg="tauv_vehicle" type="oakd_ros" name="oakd_ros" output="screen"></node>
</launch>